### ISSUE TITLE
Group chat - Group chat stacked avatars are different in Search list and LHN

### ISSUE THREAD

**Issue by kbecciv** [Expensify/App#36229]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

**Version Number:** 1.4.39.0
**Reproducible in staging?:** y
**Reproducible in production?:** y
**If this was caught during regression testing, add the test name, ID and link from TestRail:**
**Email or phone of affected tester (no customers):**
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Expensify/Expensify Issue URL:**
**Issue reported by:** Applause - Internal Team
**Slack conversation:**

## Action Performed:
1. Go to staging.new.expensify.com
2. Create a group chat.
3. Open the search list.

## Expected Result:
Group chat stacked avatars are the same in Search list and LHN.

## Actual Result:
Group chat stacked avatars are different in Search list and LHN.

## Workaround:
Unknown

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android: Native
- [ ] Android: mWeb Chrome
- [ ] iOS: Native
- [ ] iOS: mWeb Safari
- [ ] MacOS: Chrome / Safari
- [ ] MacOS: Desktop

## Screenshots/Videos

Add any screenshot/video evidence

![image](https://github.com/Expensify/App/assets/93399543/187cf256-edc8-4dbf-bbdc-c7a8279716f0)

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by kbecciv** [comment]
We think that this bug might be related to  #vip-vsb
CC @quinthar

**Comment by trjExpensify** [comment]
Yeah, I agree this is a bug. That said, @marcaaron @JmillsExpensify - how far off are we on the new group chat icons?

**Comment by mollfpr** [comment]
>To fix this issue we can make uniformity between these screens

@ZhenjaHorbach What kind of uniformity do you plan? Are these make a new component that control `SubscriptAvatar` and `MultipleAvatars`?

**Comment by mollfpr** [comment]
We only have 1 place that not showing the `SubscriptAvatar` and `MultipleAvatars` together, so I'm okay with updating just for the search page.

@trjExpensify Do we need to get the answer first or I can just review the proposal?

**Comment by trjExpensify** [comment]
You can review the proposal!

**Comment by mollfpr** [comment]
The [proposal](https://github.com/Expensify/App/issues/36229#issuecomment-1936056808) from @ZhenjaHorbach looks good to me!

🎀 👀 🎀 C+ reviewed!

**Comment by marcaaron** [comment]
>Yeah, I agree this is a bug. That said, @marcaaron @JmillsExpensify - how far off are we on the new group chat icons?

We'll add the default avatars in the first front end PR (which is not in progress yet). That'll come before the ability to add a custom avatar so it's one of the first changes we'll make.

**Comment by mollfpr** [comment]
@trjExpensify This issue missing the label `Awaiting Payment` and the payment date. Could you update it? Also, the blocker issue seems not from the PR, context: https://github.com/Expensify/App/issues/37109#issuecomment-1960514918

**Comment by trjExpensify** [comment]
Agreed. Updated!

**Comment by trjExpensify** [comment]
Payment summary as follows: 

@ZhenjaHorbach - $500 for the fix
@mollfpr - $500 for the C+ review

Paid @ZhenjaHorbach. 
@mollfpr please confirm when you've requested and we can close this out.

**Comment by mollfpr** [comment]
>[@mollfpr] The PR that introduced the bug has been identified. Link to the PR:
 [@mollfpr] The offending PR has been commented on, pointing out the bug it caused and why, so the author and reviewers can learn from the mistake. Link to comment:

No offending PR was found.

>  [@mollfpr] A discussion in #expensify-bugs has been started about whether any other steps should be taken (e.g. updating the PR review checklist) in order to catch this type of bug sooner. Link to discussion:

I think the regression step should be enough.

> [@mollfpr] Determine if we should create a regression test for this bug.
 [@mollfpr] If we decide to create a [regression test](https://github.com/Expensify/App/blob/main/contributingGuides/REGRESSION_TEST_BEST_PRACTICES.md) for the bug, please propose the regression test steps to ensure the same bug will not reach production again.

1. Open app
2. Create a group chat.
3. Open the search list.
4. Verify that avatars on the Search list item look the same as in LHN and Header chat
5. 👍 or 👎

**Comment by mollfpr** [comment]
@trjExpensify Requested on the NewDot!

**Comment by JmillsExpensify** [comment]
$500 approved for @mollfpr based on [this summary](https://github.com/Expensify/App/issues/36229#issuecomment-1976187069).
